pages/price-feeds/error-codes/evm.mdx

```diff
@@ -4,6 +4,8 @@ The following table contains the errors used in the Pyth Network's [EVM contract
 This information is derived from [PythErrors.sol](https://github.com/pyth-network/pyth-crosschain/blob/a888ba318c0325c29070eaf5afcc3a4d443b058c/target_chains/ethereum/sdk/solidity/PythErrors.sol#L8)
 in the Pyth SDK and can be used to decode error codes programmatically.
 
+Check [Troubleshoot Errors on EVM Price Feeds Contract](../troubleshoot/evm) for more information on how to handle these errors.
```

Comment:
nitpick: I think "Please see" or "Consult" would be better than "Check" in this sentence